### DISCUSSION TITLE
https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F%EF%BC%9AVisual-Studio-Code/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#19] [General]
# https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F%EF%BC%9AVisual-Studio-Code/

Back to the rough ground!

https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F%EF%BC%9AVisual-Studio-Code/

**Comment by wang7peng** [comment]


**Reply by greendolphindance** [reply]
原来如此！希望我有生之年可以用上（什么

**Comment by wang7peng** [comment]
Xcode 是为了开发开发 ios app 用的，现在只是办公做文本业务，当然毫无用武之地啦，如果哪天你学前端改学移动端开发了，还要用哦。它能绑定不同版本的 iphone 模拟器, ipadOS 模拟器，并且切换版本很方便，这些动辄 7G+，在开发运行在苹果设备上的app方面是天然优势，如果用 VS code 要鼓捣半天还不一定搞得定，也不利于团队合作。
